### PR TITLE
[CRIMAPP-504] Add trust fund page

### PR DESCRIPTION
**Pull request by hiboabd**
## Description of change
Adds trust fund page to capital flow. Also displays in review application section

## Link to relevant ticket
[CRIMAPP-585](https://dsdmoj.atlassian.net/browse/CRIMAPP-585)

### After changes:

**Form validation**
Screenshot 2024-03-12 at 09 29 51

**Scenario when yes selected and data entered**
Screenshot 2024-03-12 at 08 19 33
Screenshot 2024-03-12 at 09 30 49

**Scenario when no selected** 
Screenshot 2024-03-12 at 09 27 16
Screenshot 2024-03-12 at 09 29 36

## How to manually test the feature

Start an application and fill in the details till you get to the trust fund page in the capital assessment section
Select yes and validate the additional input fields for trust fund amount held and yearly dividend are displayed 
Then select save and continue and confirm the validation errors are displayed
Enter data into the additional fields and confirm you can save and continue to the upload evidence screen
Continue with the application to the review page and confirm you can see a trust funds section with the data matching what you entered
Check the change links take you back to the trust fund page 
Select no and save and continue 
Confirm that the review application screen displays no as the selected answer for the trust fund question

[CRIMAPP-585]: https://dsdmoj.atlassian.net/browse/CRIMAPP-585?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/presenters/summary/html_presenter.rb [hunk 1]
```diff
@@ -21,11 +21,12 @@ class HtmlPresenter
         other_income_details
         housing_payments
         other_outgoings_details
+        properties
         savings
         premium_bonds
         national_savings_certificates
         investments
-        properties
```
Comment (timpeat) [inline app/presenters/summary/html_presenter.rb:28]: Did you mean to remove properties?

File: app/models/capital.rb [hunk 2]
```diff
@@ -1,7 +1,8 @@
 class Capital < ApplicationRecord
-  attribute :premium_bonds_total_value, :pence
-
   belongs_to :crime_application
+  attribute :premium_bonds_total_value, :pence
+  attribute :trust_fund_amount_held, :pence
+  attribute :yearly_dividend, :pence
```
Comment (timpeat) [inline app/models/capital.rb:5]: I wonder if this attribute needs to be trust_fund_yearly_dividend as it is stored on capital and other things could have a yearly_dividend?

File: db/migrate/20240311151946_add_trust_fund_to_capital.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class AddTrustFundToCapital < ActiveRecord::Migration[7.0]
+  def change
+    add_column :capitals, :will_benefit_from_trust_fund, :string
+    add_column :capitals, :trust_fund_amount_held, :integer
+    add_column :capitals, :yearly_dividend, :integer
```
Comment (timpeat) [inline db/migrate/20240311151946_add_trust_fund_to_capital.rb:5]: Should this be renamed trust_fund_yearly_dividend as it is stored on capital and other things could have a yearly_dividend?

**Review by timpeat — COMMENTED** [review]
Looking great, two minor comments.

File: app/presenters/summary/html_presenter.rb (hunk 1 above)
Comment (hiboabd) [reply app/presenters/summary/html_presenter.rb:28]: I moved it as it's meant to display above savings

**Review by timpeat — APPROVED** [review]

**Pull request merged**